### ISSUE TITLE
Add elements to Mehrwert-Partner entries

### ISSUE THREAD

**Issue by sonja-retzlaff** [future-coop/transition-map#4]
In edit entry mode it should be possible to click a checkbox "Dieser Eintrag ist Transition Mehrwert-Partner" to categorize entry as Transition "Mehrwert-Partner".  When the checkbox is selected, two elements are triggered for the user frontend of the entry:

1. Hashtag #mehrwert-partner is displayed/marked in transition red (HEX #d15962)

2. The following text is displayed below description text: "Mitglieder von Transition München (https://transition-muc.de/) erhalten 5% Ermäßigung auf jeden Einkauf."

![mehrwert checkbox edit mode](https://user-images.githubusercontent.com/47123520/52267988-b0a5ab00-293a-11e9-8d4c-e49542d0d202.png)

![mehrwert partner hashtag](https://user-images.githubusercontent.com/47123520/52268014-c1562100-293a-11e9-9f7f-023bf05cac95.png)

![mehrwert partner text](https://user-images.githubusercontent.com/47123520/52268052-d5018780-293a-11e9-9365-c7212cbfa716.png)

**Comment by elbotho** [comment]
I agree with all points but would skip the checkbox in the entry form, 
using the tag is enough (and easy enough for the team) and this way we don't have to change the database.

**Comment by benediktdertinger** [comment]
Addressed in https://github.com/transition/transition-map/commit/ea4a9b63f71bd670fc6ea5d886a3531d3e2149ba
